testing/live/WorkshopOptimizerPlugin/manifest.toml

```diff
@@ -1,6 +1,6 @@
 [plugin]
 repository = "https://github.com/belzaru17/WorkshopOptimizerPlugin"

```

Comment:
Change this to:
`repository = "https://github.com/belzaru17/WorkshopOptimizerPlugin.git"`